Newborn kitten sprite variants

seyriix opened the pull request:
## About the pull request
This adds new sprite variations for the newborn lifestage!

## Why this is good for ClanGen
This gives the newborn lifestage a total of 3 sprite variations, including the original sprite, matching them to most of the other lifestages.

## Proof of testing
<details>
<summary>Examples</summary>
<img width="200" height="200" alt="Featherpaw" src="https://github.com/user-attachments/assets/edfbe0cd-1c46-49ed-84aa-2ca51105bf7e" />
<img width="200" height="200" alt="Mulberryblaze" src="https://github.com/user-attachments/assets/67286e2f-0c01-4cb2-b6c4-5f568dd7cb81" />
<img width="200" height="200" alt="Pigeonember" src="https://github.com/user-attachments/assets/c40b0509-b915-4902-94a0-e07376bbb819" />
<img width="200" height="200" alt="Scrubpelt" src="https://github.com/user-attachments/assets/a9bf44a9-ab9c-43df-8364-c190fafa1222" />
</details>

## Known issues
**bug:** cats may occasionally not display the newborn sprite correctly, defaulting to the '0' sprite. haven't encountered this in a while, but worth remembering.

## Changelog
🆑
**feat:** added new sprite variations for newborn kittens
**fix:** starfish newborn lineart has been adjusted accordingly to requested changes
**fix:** sick sprites no longer appear as the paralysed sprites
**fix:** Following white patches adjusted to better align with art style or address small mistakes:
  - EXTRA
  - BEARD
  - DIVA
  - FADESPOTS
  - MITAINE
  - RINGTAIL
  - LIGHTSONG
  - PETAL
  - MOON
  - POWDER
  - BLEACHED
  - VITILIGO
  - RAGDOLL

🆑

## Credits
Code assisted by [SabreML](https://github.com/SabreML) and [scribblecrumb](https://github.com/scribblecrumb).
Spritesheets assisted by [flamedash](https://github.com/flamedash2358), [scribblecrumb](https://github.com/scribblecrumb), and Eureka.

seyriix committed:
potatoes in progress

they're a teensy bit temperamental rn..

seyriix committed:
potatoes work?

still a wip - white and tortie patches, as well as smoke, sokoke and speckled colours, need to be finished.

seyriix committed:
changed starfish

 to match requested adjustments

seyriix committed:
lowkey forgot the lineart variants

seyriix committed:
potatoes on top!

also adjusted the code and singlecolour spritesheet to account for this.

seyriix committed:
getting through the spritesheets :)

thank you artist devs for the help!

seyriix committed:
almost all done!

thank you again artist devs for helping :D

seyriix committed:
white patches!

scribblecrumb committed:
save conversion

seyriix committed:
Merge branch 'ClanGenOfficial:development' into newborns

seyriix committed:
Merge branch 'ClanGenOfficial:development' into newborns

scribblecrumb commented:
Edited the changelog for you to include the white patch fixes i'd included along with the newborn patches ;)

also dw about the merge conflicts for now, I'll help you out with that once that other sprite PR has gotten merged, since that'll give us some more conflicts haha

seyriix committed:
potatoes all done, i think

many thanks to flamedash, scribblecrumb, and eureka from the dev team for helping complete this pr <3

scribblecrumb committed:
Merge remote-tracking branch 'upstream/development' into pr/4020

scribblecrumb committed:
updating to new json system

scribblecrumb committed:
fix newborn sprite loading

scribblecrumb committed:
Merge remote-tracking branch 'upstream/development' into pr/4020

scribblecrumb committed:
Merge branch 'development' into newborns

j-gynn reviewed (approved):
potat potat potat 🥔🥔🥔
